src/test/java/com/wikia/webdriver/testcases/adstests/TestPremiumAdLayoutFeaturedVideo.java

```diff
@@ -0,0 +1,96 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.helpers.PremiumAdLayout;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class TestPremiumAdLayoutFeaturedVideo extends TemplateNoFirstLoad {
```

Comment:
Thanks for renaming :bowing_man: What do you think about updating our [Synthetic Tests page](https://wikia-inc.atlassian.net/wiki/display/ADEN/Synthetic+tests) with it? :)